qtaf-core/pom.xml

```diff
@@ -49,12 +49,12 @@
         <log4JVersion>2.20.0</log4JVersion>
         <appiumVersion>8.3.0</appiumVersion>
         <jsonSimpleVersion>1.1.1</jsonSimpleVersion>
-        <jsonPathVersion>2.8.0</jsonPathVersion>
+        <jsonPathVersion>2.7.0</jsonPathVersion>
         <guavaVersion>31.1-jre</guavaVersion>
         <commonsLangVersion>3.12.0</commonsLangVersion>
         <reflectionsVersion>0.10.2</reflectionsVersion>
         <webDriverManagerVersion>5.3.2</webDriverManagerVersion>
-        <slf4jVersion>2.0.7</slf4jVersion>
+        <slf4jVersion>2.0.6</slf4jVersion>
```

Comment:
Is there a reason for this downgrade?

Reply:
Reverted this change, it was automatically done by the IDE